apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimResponseTransformerV2.java

```diff
@@ -56,7 +56,7 @@
             .put('d', ClaimResponse.RemittanceOutcome.COMPLETE)
             .put('i', ClaimResponse.RemittanceOutcome.PARTIAL)
             .put('r', ClaimResponse.RemittanceOutcome.COMPLETE)
-            .put('t', ClaimResponse.RemittanceOutcome.COMPLETE)
+            .put('t', ClaimResponse.RemittanceOutcome.PARTIAL)
```

Comment:
for reference, this change addresses new knowledge the team obtained from the RDA API team in July regarding the "RTP" status code.  https://confluence.cms.gov/pages/viewpage.action?pageId=455442061#comment-475157262